.github/workflows/build-bundle-linux.yml

```diff
@@ -0,0 +1,60 @@
+name: Build and Run tests
+
+on: [push]
+
+#
+# Software versions: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners#ubuntu-1804-lts
+# Help: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategy
+
+
+# env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+#   BUILD_TYPE: Release
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
We can also run it on windows and mac.

```
    strategy:
      matrix:
        platform: [ubuntu-latest, macos-latest, windows-latest]
    runs-on: ${{ matrix.platform }}
```

Reply:
Now VNote uses Travis.ci and Appyor for CI. So the GitHub way is just an alternation?

Reply:
Because cmake staff is merged bu not continuously tested, and there is better to test it without affect existing AppVeyor and Travis script.
That is why I propose github actions script.

Reply:
That would be great! When you finish it please tell me, I will merge the PR then.